packages/build-utils/src/fs/run-user-scripts.ts

```diff
@@ -748,6 +756,7 @@ export function getPathOverrideForPackageManager({
 /**
  * Helper to get the binary paths that link to the used package manager.
  * Note: Make sure it doesn't contain any `console.log` calls.
+ * @deprecated use `getPathOverrideForPackageManager` instead
```

Comment:
note that I didn't export `getPathOverrideForPackageManager` so either we need to export that, or better just recommend using `getEnvForPackageManager` instead.

Reply:
That is, `getPathOverrideForPackageManager` is exported from the module, but not from the `build-utils` package.

Reply:
My mistake. I meant `getEnvForPackageManager`. I updated it. https://github.com/vercel/vercel/pull/11596/commits/bd57cf391a731924aff4a5c34a4d5b19d8a2dff9